src/apps/media/src/app/components/FileModal/RenameFileModal.tsx

```diff
@@ -82,12 +86,16 @@ export const RenameFileModal: FC<Props> = ({
             },
           }}
           autoFocus
-          onFocus={(evt) => evt.target.select()}
+          onFocus={(evt: React.FocusEvent<HTMLInputElement>) =>
+            evt.target.select()
+          }
           value={renamedFilename}
-          onChange={(evt) => {
-            setRenamedFilename(evt.target.value);
+          onChange={(evt: React.ChangeEvent<HTMLInputElement>) => {
+            setRenamedFilename(evt.target.value.replace(" ", "-"));
           }}
-          onKeyPress={(event) => event.key === "Enter" && handleUpdate()}
+          onKeyPress={(event: React.KeyboardEvent<HTMLDivElement>) =>
+            event.key === "Enter" && handleUpdate()
```

Comment:
I'm actually not sure why this is being tagged as a change on this PR since this is not part on my changes. Must be something that's on master (since this branch is synced to latest master) that isn't on the dev branch since @shrunyan changed the target branch to dev.